sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVOptions.scala

```diff
@@ -246,7 +247,16 @@ class CSVOptions(
     lineSeparatorInRead.foreach { _ =>
       settings.setNormalizeLineEndingsWithinQuotes(!multiLine)
     }
+    settings.setErrorContentLength(CSVOptions.maxErrorContentLength)
 
     settings
   }
 }
+
+private[spark] object CSVOptions {
+  /**
+   * The max error content length in CSV parser/writer exception message.
+   */
+  val maxErrorContentLength = 1000
```

Comment:
I think It's fine just to have one variable within `CSVOptions` class and use it, like `inputBufferSize`.